storage/sealed_segment.go

```diff
@@ -68,9 +70,23 @@ const (
 	flushFailed
 )
 
+type sealedFlushingSegmentMetrics struct {
+	unload instrument.MethodMetrics
```

Comment:
I think a counter should suffice here, i.e., it's useful to know when an unload happens, and how many times it happens, but not really how long it takes to unload (and fwiw unloading should be very fast so not quite worth recording its time).